spot_rl_experiments/spot_rl/llm/src/conf/config.yaml

```diff
@@ -2,4 +2,5 @@ defaults:
   - llm: openai
   - prompt: rearrange_easy_few_shot
 
+llm_type: metaailocal # it could be "openai", "metaai", "metaailocal"
```

Comment:
Do we need both `llm` & `llm_type`?

Reply:
fixing this in the second PR. now llm flag is the actual file name

Reply:
https://github.com/facebookresearch/spot-sim2real/issues/168